src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot;
+
+import com.ctre.phoenix6.CANBus;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import edu.wpi.first.math.util.Units;
+
+public class Constants {
+    public static final CANBus DRIVETRAIN_CAN_BUS = new CANBus("3663");
```

Comment:
This should remain in the robot factories. It is specific to each robot, as they might not use the same name (Though in reality we keep them the same)